WordPress/src/main/java/org/wordpress/android/ui/posts/PostUtils.java

```diff
@@ -115,4 +128,146 @@ public static void trackSavePostAnalytics(Post post, SiteModel site) {
                 // No-op
         }
     }
+
+    public static boolean isPublishable(PostModel post) {
+        return !(post.getContent().isEmpty() && post.getExcerpt().isEmpty() && post.getTitle().isEmpty());
```

Comment:
It does seem weird, but it's perfectly acceptable to upload (and publish) a post with only the excerpt field set (and absolutely nothing else). This works from the web interface too (as well as both WP.com REST and XML-RPC).

Also, our post list will display the excerpt if there is no content, so we're handling all this correctly.

IMO, we don't need to change this.